src/main/java/org/embulk/util/config/ConfigMapperFactory.java

```diff
@@ -320,6 +323,39 @@ private ObjectMapper mapperForOthers() {
         return objectMapper;
     }
 
+    private static void assertJacksonCoreVersion() {
+        if (com.fasterxml.jackson.core.json.PackageVersion.VERSION.getMajorVersion() != 2) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson 2.");
+        }
+
+        final int minor = com.fasterxml.jackson.core.json.PackageVersion.VERSION.getMinorVersion();
+        if (minor < 14 || (minor == 15 && com.fasterxml.jackson.core.json.PackageVersion.VERSION.getPatchLevel() <= 2)) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson 2.15.3 or later.");
+        }
+    }
+
+    private static void assertJacksonDataBindVersion() {
+        if (com.fasterxml.jackson.databind.cfg.PackageVersion.VERSION.getMajorVersion() != 2) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson 2.");
+        }
+
+        final int minor = com.fasterxml.jackson.databind.cfg.PackageVersion.VERSION.getMinorVersion();
+        if (minor < 14 || (minor == 15 && com.fasterxml.jackson.databind.cfg.PackageVersion.VERSION.getPatchLevel() <= 2)) {
+            throw new UnsupportedOperationException("embulk-util-config is not used with Jackson 2.15.3 or later.");
```

Comment:
What do you think about adding `databind` in the log message?
I'm wondering that line 333 and here is the same error message.

Reply:
Thanks! Sounds reasonable. Fixed it like `"embulk-util-config is not used with Jackson (jackson-databind) 2.15.3 or later."`